src/metabase/cmd.clj

```diff
@@ -37,6 +37,15 @@
    (binding [mdb/*disable-data-migrations* true]
      ((resolve 'metabase.cmd.load-from-h2/load-from-h2!) h2-connection-string))))
 
+(defn ^:command dump-to-h2
+  "Transfer data from existing database to newly created H2 DB."
+  ([h2-filename]
```

Comment:
Style thing: you shouldn't wrap the bindings+body in a list as if the fn had multiple arities if it only has a single arity